src/azure/accountStore.ts

```diff
@@ -0,0 +1,92 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import vscode = require('vscode');
+import { IAccount } from '../models/contracts/azure/accountInterfaces';
+import Constants = require('../constants/constants');
+import AzureAuth = require('@cssuh/ads-adal-library');
+
+export class AccountStore {
+    constructor(
+        private _context: vscode.ExtensionContext
+    ) { }
+
+    public getAccounts(): IAccount[] {
+        let configValues = this._context.globalState.get<IAccount[]>(Constants.configAzureAccount);
+        if (!configValues) {
+            configValues = [];
+        }
+        return configValues;
+    }
+
+    public getAccount(key: string): IAccount {
+        let account: IAccount;
+        let configValues = this._context.globalState.get<IAccount[]>(Constants.configAzureAccount);
+        if (!configValues) {
+            // Throw error message saying there are no accounts stored
+        }
+        for (let value of configValues) {
+            if (value.key.id === key) {
+                account = value;
+                break;
+            }
+        }
+        if (!account) {
+            // Throw error message saying the account was not found
+            return undefined;
+        }
+        return account;
+    }
+
+    public removeAccount(key: string): void {
```

Comment:
Do we need this function?

Reply:
I'm going to call this function when I add sign out